core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -303,7 +303,7 @@ public SystemSessionProperties(
                 booleanProperty(
                         PLAN_WITH_TABLE_NODE_PARTITIONING,
                         "Experimental: Adapt plan to pre-partitioned tables",
-                        true,
+                        featuresConfig.isPlanWithTableNodePartitioning(),
```

Comment:
why it's enabled by default and experimental at the same time?

pre-existing, but important as you're adding config and the desc is going to be copied

Reply:
I'm removing experimental prefix (since it's enabled by default from 2016)